tasks/verify-enterprise-contract.yaml

```diff
@@ -0,0 +1,127 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: verify-enterprise-contract
+  annotations:
+    tekton.dev/displayName: Verify Enterprise Contract
+    tekton.dev/pipelines.minVersion: "0.19"
+    tekton.dev/tags: cosign, chains, signature, opa
+  labels:
+    app.kubernetes.io/version: "0.1"
+
+spec:
+  description: Verify the enterprise contract is met
+  params:
+    - name: IMAGE_REF
+      type: string
+      description: Image reference to verify
+
+    - name: PUBLIC_KEY
+      type: string
+      description: >-
+        Public key used to verify signatures. Must be a valid k8s cosign
+        reference, e.g. k8s://my-space/my-secret where my-secret contains
+        the expected cosign.pub attribute.
+
+    - name: COSIGN_EXPERIMENTAL
+      type: string
+      description: Control transparency log lookups. Set to "1" to enabled it.
+      default: "0"
+
+    - name: REKOR_HOST
+      type: string
+      description: Rekor host for transparency log lookups
+      default: https://rekor.sigstore.dev
+
+    # TODO: We should probably use a bundle URL for this:
+    # https://www.openpolicyagent.org/docs/v0.12.2/bundles/#bundle-file-format
+    # Or an OCI image:
+    # https://github.com/open-policy-agent/opa/issues/1413
+    - name: POLICY_REPO
+      type: string
+      description: Git url for rego policies
+      default: https://github.com/hacbs-contract/ec-policies.git
+
+    - name: POLICY_REPO_REF
+      type: string
+      description: Git sha, branch or tag in git repo for rego policies
+      default: main
+
+    - name: EXECUTION_IMAGE
+      type: string
+      description: The image used to executed the various steps in this task
+      # TODO: What exactly is this?
+      default: appstudio-utils
+
+  results:
+    - name: OUTPUT
+      description: A list of policy violations
+    - name: PASSED
+      description: A string formatted boolean, either true or false
+
+  steps:
+    - name: verify-image-signature
+      image: $(params.EXECUTION_IMAGE)
+      command: [cosign]
```

Comment:
What are the square brackets for here?

Reply:
`command` expects a list, thus the square brackets. The [docs](https://tekton.dev/docs/pipelines/tasks/) do not explain why `command` is a list, but every example does follow the pattern of a single item in the `command` list, and other parts going into the `args` list. I tried using a string instead, and the Tekton rejects it.

Reply:
Ack, thanks.